File: Java/Program/Product.java
//deklarasi class
class Product{
    String idProduct;
    String name;
    String brand;
    Double price;

    //constructors
    //constructor lain yang terdapat parameter
    Product(String idProduct, String name, String brand, Double price){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    //get idProduct
    String getIdProduct(){
        return this.idProduct;
    }

    //set idProduct
    void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }

    //get name
    String getName(){
        return this.name;
    }

    //set name
    void setName(String name){
        this.name = name;
    }

    //get brand
    String getBrand(){
        return this.brand;
    }

    //set brand
    void setBrand(String brand){
        this.brand = brand;
    }

    //get price
    Double getPrice(){
        return this.price;
    }

    //set price
    void setPrice(Double price){
        this.price = price;
    }
}

File: CPP/Program/Product.cpp
/*
Saya Jasmine Noor Fawzia [2200598] mengerjakan soal LP2 dalam Mata Kuliah DPBO
untuk keberkahan-Nya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan
Aamiin
*/

//import library
#include <iostream>
#include <string>
#include <iomanip>

//menggunakan standard namespace
using namespace std;

//deklarasi class
class Product{
    public:
        string idProduct;
        string name;
        string brand;
        double price;

        Product(string idProduct, string name, string brand, double price) : idProduct(idProduct), name(name), brand(brand), price(price) {}

        //get id
        string get_id(){
            return this->idProduct;
        }

        //set id
        void set_id(string idProduct){
            this->idProduct = idProduct;
        }
        
        //get name
        string get_name(){
            return this->name;
        }

        //set name
        void set_name(string name){
            this->name = name;
        }

        //get brand
        string get_brand(){
            return this->brand;
        }

        //set brand
        void set_brand(string brand){
            this->brand = brand;
        }

        //get price
        double get_price(){
            return this->price;
        }

        //set price
        void set_price(double price){
            this->price = price;
        }
    
        //destructors
        ~Product(){}
};